Views/Dashboard.js

```diff
@@ -1,27 +1,35 @@
-import React, {useEffect, useState} from 'react';
+import React, {useEffect, useState} from 'react'
 import { StyleSheet, Text, View, ScrollView } from 'react-native'
 
-//Libraries
+//LIBRARIES
 import { Button } from 'react-native-paper'
+import * as Haptics from 'expo-haptics'
+
+//COMPONENTS
+import PlantImage from '../components/ImageDisplay/PlantImage'
 import PickImage from "../components/PickImage/PickImage"
 import TakePicture from "../components/TakePicture/TakePicture"
-import * as Haptics from 'expo-haptics';
 
-export default function Dashboard({ navigation, handleChange, allImages, images, dimensions, apiKey }) {
+export default Dashboard = ({ navigation, handleChange, imageUris, imageBaseStrings, dimensions, apiKey }) => {
 
```

Comment:
One thing Im wondering is are we still using the image Base strings anymore?

Reply:
okay so what I figured out was: the image URIs are being used to display images on the phone, base strings are being sent off to get info.